packages/components/src/components/divider/divider.stories.ts

```diff
@@ -43,107 +43,72 @@ export default {
     `
   ]
 };
-/**
- * This shows sd-divider in its default state.
- */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Use the orientation attribute to set the axis of a divider.
- */
-
-export const Orientation = {
-  parameters: { controls: { exclude: 'orientation' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'orientation' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the inverted attribute to make a divider with inverted colors.
+ * You can set the component's size with `width` (horizontal) or `height` (vertical) via CSS or inline-styles.
```

Comment:
I think it would be good to keep the wording as in the other components => _"Use `attribute` to ...."_

Reply:
In this specific case, it is not an attribute, it is needed to be used other component (for example a div wrapping the divider) in order to change the size.